kuma/wiki/admin.py

```diff
@@ -476,17 +478,21 @@ class RevisionAkismetSubmissionAdmin(DisabledDeletionMixin, admin.ModelAdmin):
 
     def get_fields(self, request, obj=None):
         if obj is None:
-            return ['type', 'revision']
+            self.revision_id = request.GET.get('revision')
```

Comment:
``AdminFormWithRequest`` makes more sense now. Is that the pattern to use here as well, or do you have a better solution?